spec/classes/letsencrypt_spec.rb

```diff
@@ -141,6 +141,13 @@
           it { is_expected.to contain_file('/foo/bar/baz').with(ensure: 'directory') }
         end
 
+        describe 'with list of certificates' do
+          let(:additional_params) { { certificates: { 'foo' => {}, 'a' => {} } } }
```

Comment:
Pass in some random parameters to `foo` for instance just to check it does what is expected.